packages/relgen-core/src/services/context/remote/types.ts

```diff
@@ -55,7 +55,9 @@ export type RemoteContextService = {
       repo: string;
       num: number;
       excludedFiles?: Set<string>;
-      excludedContexts?: Set<'ticket' | 'file-content'>;
+      excludedContexts?: {
```

Comment:
The use of 'trim' is not allowed as per the rules. Please find an alternative method to handle whitespace.